api/api-roles-config-v1/src/test/groovy/com/thoughtworks/go/representers/config/rolev1/ConfigurationPropertyRepresenterTest.groovy

```diff
@@ -28,6 +29,12 @@ import static org.assertj.core.api.Assertions.assertThat
 
 class ConfigurationPropertyRepresenterTest {
 
+  @BeforeClass
+  static void init() {
+    //TODO need to find a better workaround
+    System.setProperty("json-unit.libraries", "gson");
+  }
```

Comment:
@ketan The test became flaky on my machine again, so I added this.

Reply:
Look at this http://stefanbirkner.github.io/system-rules/#EnvironmentVariables. It provides junit rules for system properties and env.